assets/src/edit-story/components/panels/design/animation/effectInput.js

```diff
@@ -56,9 +56,9 @@ const StyledInput = styled(NumericInput)`
 const StyledDropDown = styled(DropDown)`
   button {
     height: 46px;
-    ${outerGridBordersCss}
+    ${outerGridBordersCss};
     &:hover {
-      ${outerGridBordersCss}
+      ${outerGridBordersCss};
```

Comment:
Curious about adding the semi-colons to the end of these -- wouldn't this result in double semi-colons since `outerGridBordersCss` already has one as the last character?

Reply:
i was confused by a rebase and trying to figure out why linting was yelling at this, you're right this is unnecessary, i'll remove it!